website/docs/reference/node-selection/defer.md

```diff
@@ -4,6 +4,8 @@ title: "Defer"
 
 Defer is a powerful feature that makes it possible to run a subset of models or tests in a [sandbox environment](/docs/environments-in-dbt) without having to first build their upstream parents. This can save time and computational resources when you want to test a small number of models in a large project.
 
+<Lightbox src src="/img/docs/reference/defer-diagram.png" width="50%" title="This diagram explains how to defer when running model in dbt" />
```

Comment:
```suggestion
<Lightbox src src="/img/docs/reference/defer-diagram.png" width="50%" title="Use 'defer' to modify end-of-pipeline models by pointing to production models, instead of running everything upstream." />
```